autopush/integration/test/converter_test.js

```diff
@@ -9,9 +9,9 @@ describe("Color Code Converter", function() {
       var greenHex = converter.rgbToHex(0, 255, 0);
       var blueHex  = converter.rgbToHex(0, 0, 255);
 
-      expect(redHex).to.equal("ff0000");
-      expect(greenHex).to.equal("00ff00");
-      expect(blueHex).to.equal("0000ff");
+      chai.expect(redHex).to.equal("ff0000");
+      chai.expect(greenHex).to.equal("00ff00");
+      chai.expect(blueHex).to.equal("0000ff");
```

Comment:
`expect` was undefined here, so I fully qualified to `chai.expect` since `chai` is global to our HTML page (via the `<script>` tag).